sdks/python/apache_beam/io/gcsio.py

```diff
@@ -21,13 +21,13 @@
 https://github.com/GoogleCloudPlatform/appengine-gcs-client.
 """
 
+import cStringIO as StringIO
```

Comment:
I think it's standard practice to:

```python
try:
   import cStringIO as StringIO
except:
   import StringiO
```

in case of some failure modes. Is that not useful any more?